### PR TITLE
Hide screen sharing bubble on call visualizer end

### PR DESCRIPTION
**Pull request by andrews-moc**
[Hide bubble on call visualizer end](https://glia.atlassian.net/browse/MOB-1957)

[Skip survey on Call Visualizer engagement end](https://glia.atlassian.net/browse/MOB-1959)

Besides, moved several classes from `engagement` to `callvisualizer` package.

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/GliaOnCallVisualizerEndUseCase.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    interface Listener {
+        fun engagementEnded()
+    }
+
+    inner class EndCallVisualizerRunnable(private val engagement: Engagement) : Runnable {
+        override fun run() {
+            surveyRepository.onEngagementEnded(engagement)
+            if (listener != null) {
+                listener!!.engagementEnded()
+            }
```
Comment (DavDo) [inline widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/GliaOnCallVisualizerEndUseCase.kt:31]:
```suggestion
            listener?.engagementEnded()
```

Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/GliaOnCallVisualizerEndUseCase.kt:31]: Oh, yes. It's an auto-generated code. I should have changed it. Thank you.

Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/GliaOnCallVisualizerEndUseCase.kt]: Fixed dce16ed0a888aaf626697a7a745ac3a6d4fc19f2

**Review by AndriiHorishniiMOC — APPROVED** [review]

**Comment by andrews-moc** [comment]
!merge

**Pull request merged**